zerver/views/user_groups.py

```diff
@@ -121,6 +122,34 @@ def get_user_group(request: HttpRequest, user_profile: UserProfile) -> HttpRespo
     return json_success(request, data={"user_groups": user_groups})
 
 
+def are_both_setting_values_equal(
```

Comment:
I guess this is a better name for this function. We can also move this function to `zerver.lib.user_groups` probably or some file like `zerver.lib.utils`.

Reply:
`zerver.lib.user_groups` seems probably a better home.